components/ledger/ledger-utxo-flow/src/main/kotlin/net/corda/ledger/utxo/flow/impl/flows/finality/UtxoFinalityBase.kt

```diff
@@ -45,7 +45,9 @@ abstract class UtxoFinalityBase : SubFlow<UtxoSignedTransaction> {
         onFailure: ((message: String) -> Unit)? = null
     ) {
         try {
-            log.debug("Verifying signature($signature) of transaction: $transactionId")
+            log.debug {
+                "Verifying signature($signature) of transaction: $transactionId"
+            }
```

Comment:
```suggestion
            log.debug { "Verifying signature($signature) of transaction: $transactionId" }
```

Consistent formatting with debug statement 2 lines further down.